app/src/main/java/com/example/pocketcloset/Fragments/WardrobeFragment.java

```diff
@@ -206,15 +281,9 @@ public void done(List<Clothing> clothes, ParseException e) {
                     return;
                 }
 
-                // for debugging purposes let's print every post description to logcat
-                for (Clothing clothing : clothes) {
-                    //    Log.i("WardrobeFragment", "Clothing: " + clothing.getClothingType() );
-                }
-
                 // save received posts to list and notify adapter of new data
-                // adapter.clear();
+                 //adapter.clear();
                 allClothing.addAll(clothes);
```

Comment:
Consider making helper methods for mutating allClothing that also calls the necessary adapter methods. Then, you don't have to always ensure each add/remove operation always has the corresponding adapter notification.